front/components/UserMenu.tsx

```diff
@@ -114,17 +110,65 @@ export function UserMenu({
       </DropdownMenuTrigger>
 
       <DropdownMenuContent>
-        <ScrollArea className="flex max-h-96 flex-col" hideScrollBar>
-          <DropdownMenuLabel label="Beta" />
-          <DropdownMenuItem
-            label="Exploratory features"
-            icon={TestTubeIcon}
-            href={`/w/${owner.sId}/labs`}
-          />
+        <DropdownMenuLabel label="Beta" />
+        <DropdownMenuItem
+          label="Exploratory features"
+          icon={TestTubeIcon}
+          href={`/w/${owner.sId}/labs`}
+        />
+
+        <DropdownMenuLabel label="Account" />
+        <DropdownMenuItem
+          label="Profile"
+          icon={UserIcon}
+          href={`/w/${owner.sId}/me`}
+        />
+
+        <DropdownMenuItem
+          onClick={() => {
+            void router.push("/api/auth/logout");
+          }}
+          icon={LogoutIcon}
+          label="Sign&nbsp;out"
+        />
+
+        {(hasMultipleWorkspaces || showDebugTools(featureFlags)) && (
+          <DropdownMenuLabel label="Advanced" />
```

Comment:
Was this required by Design? Curious to see that changing the workspace is "Advanced" 😄

Reply:
i didn't know in which section to put it to be honest ahah, put all the submenus together + i thought that not a lot of people has multiple workspaces